### ISSUE TITLE
Add room/location data to Event list, for events that have been scheduled

### ISSUE THREAD

**Issue by jhartwin** [neinteractiveliterature/intercode#2857]

**Comment by nbudin** [comment]
Gonna kick this one to @marleighnorton for design input.  The page is already pretty cluttered IMO, so I'd like an opinion on how best to add this without making things worse.

**Comment by dkapell** [comment]
So I'm not exactly sure a) what the use case is here, vs using the 'user_signups' widget, and b) how best to represent events that have multiple runs in multiple rooms.

**Comment by jhartwin** [comment]
Use case:
kimsward 6:57 PM
On the event pages for games it  Has the time and day the games are scheduledCan we please add the Room that the game is scheduled in when you're looking at the event page
It would sure as heck make my life easier when coordinating and talking about games with people rather than having to go to the schedule and look it up

jaelen (they/them/their/themself):turtle: 7:02 PM
you mean this?
[image of event page]
If it's not showing up for a specific event, let #webteam know and/or file a github issue.

kimsward 7:20 PM
I meant when I looked at the list of events
I realize that was unclear
But its the 1st place I would go to look to see when a game was or information about it

Kim is not on github, so asked me to file the issue for them. I imagine you know how to get in touch with them, but I can also pass followup questions along to them.

**Comment by marleighnorton** [comment]
Yeah.  I'm a bit dubious about how widespread this use case is.  And still not quite clear what the use case is, honestly.  Is this as part of Kim's staff role, and the staff tools aren't good enough?

Adding more clutter to the event list makes it less useful for everyone else, so it's not a zero sum game.  I was even wondering how much sense it makes to have GMs, Authors, and Organizations.

If we really want this, I would recommend making the run start times clickable and bring up a box much like you do on the schedule page.

<img width="392" alt="Screen Shot 2019-10-25 at 5 38 39 PM" src="https://user-images.githubusercontent.com/46982612/67606085-5198f800-f74e-11e9-9b19-11618260fd06.png">

**Comment by kimsward** [comment]
The use case is to be able to quickly check what room a game is in from the list of events. There are a couple of different use case - Take a games name, find it quickly on the list of events, and see what room it is in.  The schedule is not quick for that and when looking for a game by name (which you may or may not be in) having a quick access is key, rather than having to click into the actual game.

**Comment by kimsward** [comment]
So from a scheduling perspective - it matters before con; at con - Oh I need to go find Jaelen or marleigh, they're in X game, where is that?  Or I need to run things from game Zed to Game Alpha which 2 rooms are they in.  The scrolling scheduling is not good for quick look ups like this; it scrolls and jumps far too much due to the need it has to scroll

**Comment by kimsward** [comment]
![Screenshot_20191024-192107_Chrome](https://user-images.githubusercontent.com/38927883/67607464-88710d00-f752-11e9-9881-c75664b0a610.jpg)

**Comment by kimsward** [comment]
It would also be helpful for people who need to find rooms quickly from Ops

**Comment by marleighnorton** [comment]
So if I'm understanding this correctly, when you're trying to figure out where a game is, you don't like to use the schedule by room.
https://t.interconlarp.org/events/schedule_by_room/saturday

Instead, you're going to the event list, typing in a name of the event, and checking the search results.  And you'd like the title of the game and the room to be in those search results without an extra click, yes?

**Comment by kimsward** [comment]
see the comment I just made above yours, we cross posted a bit- 

Schedule by Room is listed by Room Name not Game Name. 

On my phone the schedule is tiny and harder to read, so first I have to open it up, slide to the right time (if I've remembered the time correctly and there are instances where I might not know it when I'm looking )  and scroll up and down through the 15 or so game space options (which may become a longer list if the con rents more private rooms) to try and find the game.  

By the time I'm scrolling to the right time I'm frustrated at the website already.  I'm not sure what the problem is with adding the space under the place the time is already listed because I don't see a lot of advantage to having just 1 or the other.

**Comment by kimsward** [comment]
Okay, I can see 1 instance where you might want time and not need the room - sign ups -- except that when I do sign ups there are certain rooms I avoid due to noise issues / the air walls so I take that into consideration when looking at games as well

**Comment by kimsward** [comment]
I know that you've as a group put a lot of effort into the schedule and I don't want to besmirch that - It looks really great on desktops but it is not the most mobile friendly interface due to the sizing of text and the 2 scroll bars.  When I try to use the schedule, particularly the one with rooms,  on my cell phone ( I have a Samsung Galaxy s 7 ) it makes my eye strain, makes me disoriented when I use it. The times on the grid cut off on the top of the form when I scroll down.  I've tried it in both the horizontal and vertical phone position and had the same issues.  I started to document a scenario of a use case for using it while taking screenshots to help document what I was hitting -  trying to find games that I knew roughly when they were on the schedule for time but not where but I feel icky now so I'm going to stop.

**Comment by dkapell** [comment]
For what it's worth, i've just filed #2859 to provide the schedule in the form of a list as an additional option for mobile users, which I think addresses a number of your concerns with the schedule grid on mobile

**Comment by kimsward** [comment]
I appreciate that - but I do  think you could just fix this with the existing list -
![Capture](https://user-images.githubusercontent.com/38927883/67609619-a858fe80-f75b-11e9-84a9-8e74370aaa3e.PNG)

**Comment by dkapell** [comment]
The issue i filed would also show the events the same as they are on the schedule today, including how full they are, so it's not quite the same thing.  It doesnt adreess your desire to have rooms on the event list, but does address making the schedule easier to use on mobile

**Comment by kimsward** [comment]
so it doesn't solve my use case, it solves the problems with the mobile schedule which are separate issue that came up during this discussion
